lib/fastlane/plugin/appcenter/actions/appcenter_fetch_devices_action.rb

```diff
@@ -0,0 +1,138 @@
+          "Especially useful when combined with register_devices and match to automatically register and provision devices with Apple. " +
+          "By default, only the Collaborators group will be included, use `destination: '*'` to match all groups."
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :api_token,
+                                  env_name: "APPCENTER_API_TOKEN",
+                                  sensitive: true,
+                               description: "API Token for App Center",
+                                  optional: false,
+                                      type: String,
+                             verify_block: proc do |value|
+                               UI.user_error!("No API token for App Center given, pass using `api_token: 'token'`") unless value && !value.empty?
+                             end),
+          FastlaneCore::ConfigItem.new(key: :owner_name,
+                                  env_name: "APPCENTER_OWNER_NAME",
+                               description: "Owner name",
+                                  optional: false,
+                                      type: String,
+                              verify_block: proc do |value|
+                                UI.user_error!("No Owner name for App Center given, pass using `owner_name: 'name'`") unless value && !value.empty?
+                              end),
+          FastlaneCore::ConfigItem.new(key: :app_name,
+                                  env_name: "APPCENTER_APP_NAME",
+                               description: "App name",
+                                  optional: false,
+                                      type: String,
+                              verify_block: proc do |value|
+                                UI.user_error!("No App name given, pass using `app_name: 'app name'`") unless value && !value.empty?
+                              end),
+          FastlaneCore::ConfigItem.new(key: :devices_file,
+                                  env_name: "FL_REGISTER_DEVICES_FILE",
+                               description: "File to save devices to",
+                                      type: String,
+                             default_value: "devices.txt",
+                              verify_block: proc do |value|
+                                UI.important("Important: Devices file is #{value}. If you plan to upload this file to Apple Developer Center, the file must have the .txt extension") unless value && value.end_with?('.txt')
+                              end),
+          FastlaneCore::ConfigItem.new(key: :destinations,
+                                  env_name: "APPCENTER_DISTRIBUTE_DESTINATIONS",
+                               description: "Comma separated list of distribution group names. Default is 'Collaborators', use '*' for all distribution groups",
+                             default_value: "Collaborators",
```

Comment:
Is the default `Collaborators` instead of `*` for safety reasons? I feel pretty strongly that most people would want to fetch from all groups assigned to an app, but I'll defer to you here. I can see the safety angle because of the only yearly chance to remove registered devices.

Reply:
I'd rather be safe here. Also, we can't change the default behavior for the existing action, so for consistency as well let's use Collaborators. I've added '*' for your desired behavior :)